source/common/http/alternate_protocols_cache_manager_impl.cc

```diff
@@ -59,7 +59,7 @@ AlternateProtocolsCacheSharedPtr AlternateProtocolsCacheManagerImpl::getCache(
     const AlternateProtocolsCacheImpl::Origin origin = {"https", entry.hostname(), entry.port()};
     std::vector<AlternateProtocolsCacheImpl::AlternateProtocol> protocol = {
         {"h3", entry.hostname(), entry.port(),
-         dispatcher.timeSource().monotonicTime() + std::chrono::hours(168)}};
+          dispatcher.timeSource().monotonicTime() + std::chrono::hours(168)}};
```

Comment:
nit: this looks mis-indented, but I guess check-format is good with it? Do you understand why?